libsyclinterface/source/dpctl_sycl_queue_interface.cpp

```diff
@@ -121,7 +121,7 @@ bool set_kernel_arg(handler &cgh,
     return arg_set;
 }
 
-std::unique_ptr<property_list> create_property_list(int properties)
+static std::unique_ptr<property_list> create_property_list(int properties)
```

Comment:
Why is the function changed to `static`?

Reply:
I sued static in C sense, as this function is not public, but it is already in the anonymous namespace, so going to revert addition of `static` keyword here.